lib/vrl/stdlib/src/downcase.rs

```diff
@@ -34,6 +34,11 @@ impl Function for Downcase {
             result: Ok("foo 2 bar"),
         }]
     }
+
+    fn call(&self, _ctx: &mut Context, args: &mut VmArgumentList) -> Resolved {
+        let value = args.required("value");
+        Ok(value.try_bytes_utf8_lossy().unwrap().to_lowercase().into())
```

Comment:
Even though this is just a one-liner, I'd consider factoring out a `downcase` function similar how it's done to the others.

Otherwise this is an invitation to introduce feature drift. I'd apply the same principle to all VRL function implementations.